site/en/guide/migrate.ipynb

```diff
@@ -1773,16 +1770,19 @@
       "source": [
         "### Premade Estimators\n",
         "\n",
-        "[Premade Estimators](https://www.tensorflow.org/guide/premade_estimators) in the family of `tf.estimator.DNN*`, `tf.estimator.Linear*` and `tf.estimator.DNNLinearCombined*` are still supported in the TensorFlow 2.0 API, however, some arguments have changed:\n",
+        "[Premade Estimators](https://www.tensorflow.org/guide/premade_estimators) in the family of `tf.estimator.DNN*`, `tf.estimator.Linear*` and `tf.estimator.DNNLinearCombined*` are still supported in the TensorFlow 2.0 API. However, some arguments have changed:\n",
         "\n",
         "1. `input_layer_partitioner`: Removed in 2.0.\n",
         "2. `loss_reduction`: Updated to `tf.keras.losses.Reduction` instead of `tf.compat.v1.losses.Reduction`. Its default value is also changed to `tf.keras.losses.Reduction.SUM_OVER_BATCH_SIZE` from `tf.compat.v1.losses.Reduction.SUM`.\n",
-        "3. `optimizer`, `dnn_optimizer` and `linear_optimizer`: this arg has been updated to `tf.keras.optimizers` instead of the `tf.compat.v1.train.Optimizer`. \n",
+        "3. `optimizer`, `dnn_optimizer` and `linear_optimizer`: this argument has been updated to `tf.keras.optimizers` instead of the `tf.compat.v1.train.Optimizer`. \n",
         "\n",
         "To migrate the above changes:\n",
+        "\n",
         "1. No migration is needed for `input_layer_partitioner` since [`Distribution Strategy`](https://www.tensorflow.org/guide/distributed_training) will handle it automatically in TF 2.0.\n",
-        "2. For `loss_reduction`, check [`tf.keras.losses.Reduction`](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/losses/Reduction) for the supported options.\n",
-        "3. For `optimizer` args, if you do not pass in an `optimizer`, `dnn_optimizer` or `linear_optimizer` arg, or if you specify the `optimizer` arg as a `string` in your code, you don't need to change anything. `tf.keras.optimizers` is used by default. Otherwise, you need to update it from `tf.compat.v1.train.Optimizer` to its corresponding [`tf.keras.optimizers`](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/optimizers)\n"
+        "2. For `loss_reduction`, check `tf.keras.losses.Reduction` for the supported options.\n",
+        "3. For `optimizer` arguments: \n",
+        "    - If you do not pass in an `optimizer`, `dnn_optimizer` or `linear_optimizer` arg, or if you specify the `optimizer` argument as a `string` in your code, you don't need to change anything — `tf.keras.optimizers` is used by default. \n",
```

Comment:
```suggestion
        "    - If you do not pass in an `optimizer`, `dnn_optimizer` or `linear_optimizer` arg, or if you specify the `optimizer` argument as a `string` in your code, you don't need to change anything because `tf.keras.optimizers` is used by default. \n",
```

Reply:
@lamberta Please take a look when you can and let me know if this structure is OK 🤔